sklearn_numba_dpex/kmeans/kernels/lloyd_single_step.py

```diff
@@ -41,15 +41,30 @@ def make_lloyd_single_step_fixed_window_kernel(
     work_group_size,
     dtype,
 ):
+    # The height of the window on centroids (or, equivalently, the number of features
+    # in the window), and the width (number of centroids in the window), are chosen
+    # such that:
+    # - the width is equal to the `sub_group_size`, which is the mininal size that
+    # optimizes IO patterns by having items in each sub group read contiguous memory
+    # slots, which can optimize IO bandwitch by trigerring coalescence of the read
+    # operations of each item in the sub group. Higher sizes would increase the
+    # pressure on private memory (since the size of the private array
+    # `dot_products` is equal to the width of the window) with no particular gain to
+    # expect.
```

Comment:
```suggestion
    #   - the width is equal to the `sub_group_size`, which is the minimal
    #   size that optimizes IO patterns by having items in each sub group
    #   read contiguous memory slots.  This configuration can optimize IO
    #   bandwidth by triggering coalescence of the read operations of each
    #   item in the sub group. Higher sizes would increase the pressure on
    #   private memory (since the size of the private `dot_products` array
    #   is equal to the width of the window) with no particular gain to expect.
```

Reply:
Actually it would be even better to indent the lines within a bullet point to match the column of the first non empty char after the `-` to match Restructured Text / markdown conventions.

Reply:
Done.

Reply:
Is this diff only formating or is there another change I miss ? why would we enforce 80 characters limits on the inline comments while the character limit for the code is above this threshold (88 IIRC) ?

Reply:
I should have precised the nature of this suggestion.

This suggestion proposes rephrasing this paragraph (it especially proposes breaking long lines) and does some reformatting of the comment.

A 88 (or even 80) characters' limit is good to me, but I have not checked whether this comment currently complies with such limit.